rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProviderImpl.java

```diff
@@ -385,6 +398,18 @@ private void saveProposedFederation(ActivationConfig.ForBlock activations) {
         bridgeStorageAccessor.saveToRepository(PROPOSED_FEDERATION.getKey(), proposedFederation, BridgeSerializationUtils::serializeFederation);
     }
 
+    private void saveFundTransactionUnsignedHash(ActivationConfig.ForBlock activations) {
+
+        if (!isFundTransactionUnsignedHashSet || !activations.isActive(RSKIP419)) {
```

Comment:
please put the same order as in [here](https://github.com/rsksmart/rskj/pull/2651/files#diff-0973099a21ca000a1f5c207a9b101cb5bc685d4cde430ce8432f12af15a220b0R392)

Reply:
Updated.